scripts/builtin/scale.dml

```diff
@@ -0,0 +1,42 @@
+#-------------------------------------------------------------
+#
+# Copyright 2018 Graz University of Technology
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#-------------------------------------------------------------
+
+m_scale = function(Matrix[Double] X, Boolean center, Boolean scale) return (Matrix[Double] Y) {
+    # This function centers scales and performs z-score on the input matrix X
+    Y = X;                  # return as is if both center and scale are FALSE
+    if (center == TRUE) {
```

Comment:
For all, boolean variables and predicates (3 times), simplify `if( center == TRUE )` to `if( center )`